docs/design/README.md

```diff
@@ -2226,18 +2226,21 @@ being passed as a separate explicit argument.
 > -   Proposal
 >     [#950: Generic details 6: remove facets](https://github.com/carbon-language/carbon-lang/pull/950)
 
-### Checked and template parameters
+### Checked and unchecked parameters
 
-The `:!` indicates that `T` is a _checked_ parameter passed at compile time.
-"Checked" here means that the body of `Min` is type checked when the function is
+The `:!` indicates that `T` is a type that is bound at compile time. There are
```

Comment:
`:!` parameters need not be types, we also support non-type compile-time parameters, like C++'s [non-type template parameters](https://en.cppreference.com/w/cpp/language/template_parameters#Non-type_template_parameter).

Reply:
I can change "type" to "parameter"?